index.html

```diff
@@ -274,6 +275,43 @@ <h1 class="underlined">About Me</h1>
                 </ul>
             </div>
         </section>
+
+        <!-- contact me  -->
+        <section class="get-in-touch">
+            <p>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+            <form action="https://formspree.io/f/xyyawvzk" enctype="multipart/form-data" method="POST"
+                class="get-in-touch-form">
+                <div class="name-input">
+                    <input type="text" name="first name" placeholder="First Name" maxlength="30" required>
+                    <input class="last-name-input" type="text" name="last name" placeholder="Last Name" maxlength="30"
+                        required>
+                </div><input type="email" name="email" placeholder="Email Address" required>
+                <textarea name="message" id="message" cols="30" rows="5" maxlength="500" required
+                    placeholder="Enter text here">
+                </textarea>
+                <button type="submit">Get in touch</button>
```

Comment:
- [ ] Kindly, add button interaction like `hover` `active` `enabled`. This will make you project more interacctive and user friendly 📱😄
![image](https://user-images.githubusercontent.com/74411135/228925153-87d485a0-48b0-4308-8eeb-5794e68f38a1.png)